google/resource_dataproc_cluster.go

```diff
@@ -802,7 +802,7 @@ func flattenPreemptibleInstanceGroupConfig(d *schema.ResourceData, icg *dataproc
 func flattenInstanceGroupConfig(d *schema.ResourceData, icg *dataproc.InstanceGroupConfig) []map[string]interface{} {
 	disk := map[string]interface{}{}
 	data := map[string]interface{}{
-	//"instance_names": []string{},
+		//"instance_names": []string{},
```

Comment:
ugh this is because I'm on go version 1.10. We should probably standardise on one of them at some point -_-

Reply:
Ugh yeah I just upgraded and hit this- let's just remove that comment (I'm not even sure why it's there) in a separate PR